tpmutil/tbs/tbs_windows.go

```diff
@@ -189,17 +197,17 @@ func (context Context) SubmitCommand(
 		uintptr(context),
 		uintptr(commandLocalityZero),
 		uintptr(priority),
-		uintptr(unsafe.Pointer(&(commandBuffer[0]))),
+		sliceAddress(commandBuffer),
```

Comment:
Is there any command where passing nil commandBuffer is valid?
If not, I'd rather check that before calling the command and return the error directly

Reply:
The comments and tests should clarify what happens when any of the slices in question are nil. I think it's fine for us to just pass along whatever buffer we are given, and let the syscall do the error checking. The TBS library is supposed to just be a safe wrapper around the DLL.